requirements.txt

```diff
@@ -1,62 +1,62 @@
 -e git+https://github.com/habnabit/txstatsd.git@157ef85fbdeafe23865c7c4e176237ffcb3c3f1f#egg=txStatsD-master
 apns==2.0.1
-asn1crypto==0.24.0        # via cryptography, treq
```

Comment:
How come we're not mentioning treq anymore?

Reply:
I did notice this oddity and double checked [treq's dependencies](https://github.com/twisted/treq/blob/master/setup.py), which seem to line up with this latest change. I suspect the version of pip upgrade JR used last week went rogue or something..